syncopy/tests/synth_data.py

```diff
@@ -101,7 +120,8 @@ def phase_diffusion(freq,
                     fs=1000,
```

Comment:
This should have been `samplerate` right from the 1st implementation, before merge I would push a little commit adressing this